saltproc/openmc_depcode.py

```diff
@@ -125,34 +125,31 @@ def read_depleted_materials(self, read_at_end=False):
 
         """
 
-    def run_depletion_step(self, cores, nodes):
-        """Runs a depletion step in OpenMC as a subprocess with the given
-        parameters.
+    def run_depletion_step(self, mpi_args=None, threads=None):
```

Comment:
I don't see `threads` called in this function, is it needed here?

Reply:
It's needed in the signature, but it isn't used in the function.